assets/section-main-product.css

```diff
@@ -501,6 +506,10 @@ a.product__text {
   .product__media-list .product__media-item {
     width: calc(100% - 3rem - var(--grid-mobile-horizontal-spacing));
   }
+
+  .product--mobile-columns .product__media-item {
+    width: calc(50% - 1.5rem - var(--grid-mobile-horizontal-spacing));
+  }
```

Comment:
Thanks for the feedback, Ken!

> I used `1.5rem` for the following CSS for `product__media-item` because I think it provides much better spacing compared to `3rem`: [before](https://user-images.githubusercontent.com/28404165/183519257-cdf78086-385c-495c-ba53-e25aa3b12b21.png) / [after](https://user-images.githubusercontent.com/28404165/183519334-339a0fe3-1f6c-4e02-9f80-646306fff94f.png)

To provide more context as to why I chose to deviate from our existing `3rem` value was mainly due to how 'things' didn't seem to provide a consistent 'left' and 'right' peek (_if you will_) -- here's a video that shows what I'm referring to:

https://user-images.githubusercontent.com/28404165/186271121-fea437f2-2384-4716-8e1d-58a65ef619a3.mp4

I'm more than happy to change this. And good call on seeing how UX feels about this -- @YoannJailin please let us know your thoughts and whether I should use the existing `3rem` width or the proposed, `1rem` width shown in the video above.

Reply:
Thanks Olivia!